docker/datahub-frontend/Dockerfile

```diff
@@ -52,4 +52,4 @@ ENV JAVA_OPTS=" \
    -Dlogback.configurationFile=datahub-frontend/conf/logback.xml \
    -Dlogback.debug=false \
    -Dpidfile.path=/dev/null"
-CMD ["datahub-frontend/bin/playBinary"]
+CMD ["datahub-frontend/bin/datahub-frontend"]
```

Comment:
Assuming you've verified that building and running the container locally works as expected

Reply:
Yeah, I'll double check with the one that gets published though just to make sure I'm not pulling in any local state.